tests/Computation.Tests/NormalFloatTests.cs

```diff
@@ -4,11 +4,11 @@
 using Symbolica.Expression;
 using Xunit;
 
-namespace Symbolica.Computation.Values;
+namespace Symbolica.Computation;
 
 public class NormalFloatTests
 {
-    private static readonly IContextFactory ContextFactory = new SharedContextFactory();
```

Comment:
How come we don't need to use `SharedContextFactory` here anymore?

Reply:
We never really needed it here since the 10s of `Context`s used by these particular tests would have been fine anyway. The reason we needed it to exist at all was that the 1000s of `Context`s needed for all the other tests caused memory issues. So it was only really used here because they were all the same pattern.

I can get rid of this thing entirely (within this test project) now because I have moved the tests from the wrapper `Expression` layer to each `IValue` implementation instead. This means we can control the instantiation of `Context` ourselves, and just use a single static one per test class. This still results in 10s of `Context`s in total, which is fine from a memory point of view.

Unfortunately we do still need the pattern of allowing injection of the `IContextFactory` implementations, rather than just allowing `Computation` to construct `Context`s whenever it needs one, because the `Application` integration tests still has memory issues (since the native dependency is not unloaded properly between runs). Annoying.